pkg/commands/clusterstack/create.go

```diff
@@ -54,6 +53,20 @@ kp clusterstack create my-stack --build-image ../path/to/build.tar --run-image .
 	return cmd
 }
 
+func ConfigureFactory(factory *clusterstack.Factory, ch *commands.CommandHelper) {
```

Comment:
Is this `ConfigureFactory` exported?

Reply:
Would there be a way to avoid the need for this ConfigureFactory setup?

Could we still inject a Relocator delegate that is configured on a PreHook on the rootcommand?

Reply:
> Could we still inject a Relocator delegate that is configured on a PreHook on the rootcommand?

have to also consider the fact that the tests for these commands inject the fakes during command construction. 

for the record, I did consider the delegate option as done for the store delete command - confirmation provider, but felt like a no-go

Reply:
I guess the PreRun hook wouldn't be active for the tests.. will look more into your suggestion.

Reply:
addressed in commit https://github.com/vmware-tanzu/kpack-cli/commit/be211733ecaa33231703f7b9d2938b0b71bee3a3

Reply:
marking as resolved. addressed in commit https://github.com/vmware-tanzu/kpack-cli/commit/d8e823f0700a84cf12ca603965271252fad97e27
please reopen if not ok